apps/src/templates/instructions/codeReviewV2/CodeReviewDataApi.js

```diff
@@ -135,4 +193,48 @@ export default class CodeReviewDataApi {
       }
     ];
   }
+
+  submitNewCodeReviewComment(commentText) {
+    return new Promise((resolve, reject) => {
+      findProfanity(commentText, appOptions.locale, this.token)
```

Comment:
Oh interesting, didn't realize we did this client side too.  I think we'll also need to do this on the back-end?  😬

Reply:
Huh - yeah I just pulled this from the last version of code review, it would be safer to also do it on the back-end..